nominal/nominal.py

```diff
@@ -635,3 +635,33 @@ def get_data_review(rid: str) -> DataReview:
     """Retrieve a data review from the Nominal platform by its RID."""
     conn = get_default_client()
     return conn.get_data_review(rid)
+
+
+def upload_journal_json(
```

Comment:
I'd probs remove this for now (I don't want to duplicate every single thing in this file)